src/home/spaces_dock.rs

```diff
@@ -88,6 +91,30 @@ live_design! {
         }
     }
 
+    Logout = {{Logout}} {
```

Comment:
this is a vague name, it's a button so call it that.
```suggestion
    LogoutButton = {{LogoutButton}} {
```
or `LogoutButtonWrapper` if you wish.